src/window/mod.rs

```diff
@@ -402,8 +411,16 @@ pub fn create_window() {
 
     log::trace!("repositioned window: {}", did_reposition);
 
+    let settings = SETTINGS.get::<WindowSettings>();
+    let window_padding = WindowPadding {
+        top: settings.top_padding,
+        left: settings.left_padding,
+        right: settings.right_padding,
+        bottom: settings.bottom_padding,
+    };
+
     let scale_factor = windowed_context.window().scale_factor();
-    let renderer = Renderer::new(scale_factor);
+    let renderer = Renderer::new(scale_factor, window_padding);
```

Comment:
Since you already have the individual settings, putting `WindowPadding` as a whole into the settings and implementing `ParseFromValue` on it would be cleaner, I guess. Or you _directly_ query the settings already in `Renderer::new`. But creating it externally from an unrelated context and passing it in is a bit douple-hooped, imo.